src/components/myAccount/editProfileInfo.component.js

```diff
@@ -5,21 +5,12 @@ import Button from "@material-ui/core/Button";
 import { useSelector, useDispatch } from "react-redux";
```

Comment:
Doesn't look like we're using `withStyles` and `Button` on lines 3 and 4 so we can probably comment those lines out or get rid of them for now